pogobuf/lehmer.js

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
Hmmm. Maybe it should go in a new folder `lib` then. Currently in the `pogobuf` folder we only have the pogobuf files itself. Or maybe it can be merged into `pogobuf.utils.js`?

Or maybe you can just turn it into an npm module? 😃